### PR TITLE
add a doc for explaining details about training data

### PR DESCRIPTION
**Pull request by yjchoi1**
**Describe the PR**
Added doc for explaining details about training data

**Related Issues/PRs**

**Additional context**

### CODE REVIEW

**Review by kks32 — CHANGES_REQUESTED** [review]
Starting with Python 3.10, ragged version of arrays is not supported. How should we change the format so that we can continue to support `npz` formats?

**Comment by yjchoi1** [comment]
The current data structure used in meshnet (dictionary-based) works fine with the up-to-date numpy and python 3.10.

File: docs/flow_data.md [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+    "simulation_1": {
+        ...
+    },
+    "simulation_n": {
+        ...
+    }
+}
+```
+
+# About `pos`
```
Comment (kks32) [inline docs/flow_data.md:27]: These should be subheadings. 
```suggestion
## `pos`
```

File: docs/flow_data.md [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+
+       [[px0_t99, py0_t99],  # at t=99
+        [px1_t99, py1_t99],
+        ...,
+        [px16_t99, py16_t99]
+        [px17_t99, py17_t99]]], shape=(100, 18, 2))
+```
+
+
+# About `node_type`
```
Comment (kks32) [inline docs/flow_data.md:57]:
```suggestion
## `node_type`
```

File: docs/flow_data.md [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+       ...
+
+       [[vx0_t99, vy0_t99],  # at t=99
+        [vx1_t99, vy1_t99],
+        ...,
+        [vx16_t99, vy16_t99]
+        [vx17_t99, vy17_t99]]], shape=(100, 18, 2))
+```
+
+# About `cells`
```
Comment (kks32) [inline docs/flow_data.md:124]:
```suggestion
## `cells`
```

**Review by kks32 — CHANGES_REQUESTED** [review]

**Comment by yjchoi1** [comment]
The suggestion has been committed.

**Review by kks32 — APPROVED** [review]

**Pull request merged**